pkg/querier/blocks_consistency_checker.go

```diff
@@ -45,13 +45,13 @@ func NewBlocksConsistencyChecker(uploadGracePeriod, deletionGracePeriod time.Dur
 func (c *BlocksConsistencyChecker) Check(knownBlocks bucketindex.Blocks, knownDeletionMarks map[ulid.ULID]*bucketindex.BlockDeletionMark, queriedBlocks []ulid.ULID) (missingBlocks []ulid.ULID) {
 	c.checksTotal.Inc()
 
-	// Reverse the map of queried blocks, so that we can easily look for missing ones.
```

Comment:
I don't think this comment is quite true, because `queriedBlocks` is an array and not a map.